primitive/compute/description/preprocessing.go

```diff
@@ -350,12 +350,6 @@ func createFilterData(filters []*model.Filter, columnIndices map[string]int, off
 			wrapper := NewDatasetWrapperStep(map[string]DataRef{"inputs": &StepDataRef{offset - 1, "produce"}}, []string{"produce"}, offset, "")
 			filterSteps = append(filterSteps, filter, wrapper)
 			offset += 2
-
-		case model.FeatureFilter, model.TextFilter:
```

Comment:
I'm a bit worried that text filter is also being dropped. Do we never use it here?

Reply:
That shouldn't have been removed.  I'll put it back in.